apis/Google.Cloud.Kms.V1/Google.Cloud.Kms.V1/Google.Cloud.Kms.V1.csproj

```diff
@@ -25,7 +25,7 @@
     <PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="1.0.1" PrivateAssets="All" />
     <PackageReference Include="Google.Api.Gax.Grpc" Version="2.7.0" />
     <PackageReference Include="Google.Cloud.Iam.V1" Version="1.1.0" />
-    <PackageReference Include="Grpc.Core" Version="1.18.0" PrivateAssets="None" />
+    <PackageReference Include="Grpc.Core" Version="1.20.1" PrivateAssets="None" />
```

Comment:
Why not version 1.21.0?

Reply:
Because the default specified by the project generator is 1.20.1. We can update that in the future of course.